pages/_app.js

```diff
@@ -43,13 +44,51 @@ export default function App({ Component, pageProps }) {
     );
   }
 
+  function handleAddComments(slug, comment) {
```

Comment:
```suggestion
  function handleAddComment(slug, comment) {
```
Should be in singular, as you pass the function as `onSubmitComment`